### ISSUE TITLE
Warn for incorrect use of slash '/' when backslash '\' for line-wrap is most likely intended

### ISSUE THREAD

**Issue by TinCanTech** [koalaman/shellcheck#2227]
#### For bugs
- Rule Id (if any, e.g. SC1000): 
- My shellcheck version (`shellcheck --version` or "online"): 
- [X] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [X] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [X] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [X] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/sh

test_file='foo.bar'
[ -f "$test_file" ] || /
	echo error!
```

#### Here's what shellcheck currently says:
$ shellcheck myscript
No issues detected!

#### Here's what I wanted or expected to see:
The use case above is for wrapping long lines with `\` but I incorrectly used `/` and `shellcheck` accepted it.

In `/bin/sh` this is an error:
```
$ / || echo $? 
sh: 1: /: Permission denied
126
```
AFAIK, in order to use `\` for line wrap the `\` MUST be the last character on the line, so it seems like a reasonably easy check for `shellcheck` to detect.  I may be wrong, of course ..

`shellcheck` should at least warn for this because it took me an hour to spot my stupid mistake.

Thanks ;-)

**Comment by koalaman** [comment]
This is now highlighted as part of a general command name sanity check that warns about commands ending in `/` (and other symbols). Thanks!

**Comment by TinCanTech** [comment]
Marvellous .. and thank you  :smile:
